content/README.md

```diff
@@ -0,0 +1,11 @@
+# Content Structure
+
+In this folder you will find content for the following:
+- meetings
+- talks
+- speakers
+- hosts
+- sponsors
```

Comment:
could be useful to make these hyperlinks to the actual directories

Reply:
I agree, thats a good idea. We can fix that up as a follow up if you’re fine with it. 

Where should we park the todo, in the issue or on the projects board?

Reply:
i tend to start with a card on the project but immediately create an issue from it. unnecessary if it wont be out there long tho